libraries/PPP/src/ppp.c

```diff
@@ -1,3 +1,4 @@
+#if CONFIG_LWIP_PPP_SUPPORT
```

Comment:
add `#include "sdkconfig.h"` before this line

Reply:
yeah, needed. Did work since compiled in Arduino Lib Builder...